lms/envs/aws.py

```diff
@@ -176,7 +176,11 @@
 EDXMKTG_USER_INFO_COOKIE_NAME = ENV_TOKENS.get('EDXMKTG_USER_INFO_COOKIE_NAME', EDXMKTG_USER_INFO_COOKIE_NAME)
 
 LMS_ROOT_URL = ENV_TOKENS.get('LMS_ROOT_URL')
```

Comment:
The reason I did it this way is because the line above appears to be very intentional. An error will be explicitly thrown during Django startup when running in production if `LMS_ROOT_URL` is set to `None`: https://github.com/edx/edx-platform/blob/master/common/djangoapps/startup_configurations/validate_config.py#L31

Also, with this change, the `ENTERPRISE_API_URL` will never be `None` for a successfully started LMS/CMS because:
- When running with devstack/docker_devstack/test settings, this value is overridden again, just like `LMS_ROOT_URL` is.
- When running with aws/openstack settings, Django will fail during setup because `LMS_ROOT_URL` *must* be explicitly set (as has always been the case).